policy/modules/admin/brctl.te

```diff
@@ -17,12 +17,13 @@ role brctl_roles types brctl_t;
 # Local policy
 #
 
-allow brctl_t self:capability net_admin;
+allow brctl_t self:capability { net_admin };
 allow brctl_t self:fifo_file rw_fifo_file_perms;
 allow brctl_t self:unix_stream_socket create_stream_socket_perms;
 allow brctl_t self:unix_dgram_socket create_socket_perms;
 allow brctl_t self:tcp_socket create_socket_perms;
 
+kernel_load_module(brctl_t)
```

Comment:
Can this be dontaudited instead? I would hope the below module load request would be sufficient.

Reply:
I'll try that